matchmaking/backend/controllers/journey.js

```diff
@@ -0,0 +1,18 @@
+const { journeyQueries } = require('../queries');
+
+/**
+ * Get journey by user id.
+ * @author Brandon Bouchard <brandon.bouchard@gov.bc.ca | brandonjbouchard@gmail.com>
+ * @method GET
+ * @param {string} id - User GUID, a unique identifier for a user.
+ */
+exports.get_journey = async (req, res) => {
+  try {
+    const journey = await journeyQueries.getJourneyByUserId(req.params.id);
+
+    if (journey) res.status(200).json(journey); // Success, return journey.
```

Comment:
were you doing pseudo here? I'd love to talk about how we structure commenting later as a group

Reply:
Whats wrong w the comments?